cmake/AwsTestHarness.cmake

```diff
@@ -87,6 +90,7 @@ function(generate_cpp_test_driver driver_exe_name)
 
     foreach(name IN LISTS TEST_CASES)
         add_test(${name} ${driver_exe_name} "${name}")
+        set_tests_properties("${name}" PROPERTIES SKIP_RETURN_CODE ${SKIP_RETURN_CODE_VALUE})
```

Comment:
do we need this if harness is checking return code of the test point?

Reply:
Yes. The ctest still need to know which error code is defined as "skipped" from the harness result.